docs/moment/04-displaying/01-format.md

```diff
@@ -409,8 +404,6 @@ If you are more comfortable working with strftime instead of LDML-like parsing t
 
 #### Default format
 
-As of version **1.5.0**, calling `moment#format` without a format will default to `moment.defaultFormat`. Out of the box, `moment.defaultFormat` is the ISO8601 format `YYYY-MM-DDTHH:mm:ssZ`.
```

Comment:
We should leave this, no? We can remove the reference to the version if you like.